src/content/en/fundamentals/primers/service-workers/the-players.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: shared/narrow
+published: false
+title: "The Players"
+description: "A basic service worker implementation has two types of players."
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 3
+translation_priority: 0
+authors:
+- josephmedley
+---
+
+<p class="intro">
+  A basic service worker implementation has two types of players.
+</p>
+
+ 
+### Clients
+
+These players install and use a service worker. Generally, they  are web
+pages. Though there are [other types of clients](https://developer.mozilla.org/en-US/docs/Web/API/Client)
+,  this primer focuses on web page clients.
+
+### Service worker script
+
+This player is a background script that acts as a network proxy for one or
+more clients. Though its name implies activity, you can almost think of a
+service worker as a passive player, sitting around waiting for its events to
```

Comment:
"sitting around waiting for its events to be called" makes it sound like the service worker is always running.

Instead, could you make it clear here that the browser stops service workers after a period of idleness, and then starts them up again when it's time to handle an event?